chore: Create web_component.md (#4889)
swisspost/design-system: .github/ISSUE_TEMPLATE/web_component.md

---
name: Web Component

> about: Parent issue for adding sub-issues related to this component.

Reviewer: The web component is already a sub-issue of the base component, right?

Author: Yeah... copy & paste error ^^

> name: Web Component
> about: Parent issue for adding sub-issues related to this component.

Author: ```suggestion
name: Component/Web Component
about: For creating a web component implementation.
```

> title: ''

Reviewer: Same thing here, maybe a `Component name [Web Component]`?

Author: I'm not sure about this. For the bug template for example, the default title is "[bug]: ". When creating issues, it's very easy to overlook that the title needs to be set as well. As it already has some content, the field is validated and the issue is created with just the title "[bug]: ". If the title is empty from the start, it's impossible to forget to put one.

Also, since labels and issue type are already set, I was hoping this would be enough to distinguish the different types of tickets and we would not need an additional distinction in the title. What do you think?

Reviewer: I think visually, it does help to have the `[type]:` before name of the component, as it's currently the case for `[bug]` and `[component]` tickets now. I think Sandra had also made a comment last time we discussed it that it helped her as well. Maybe it's something we could discuss tomorrow with the team, to have their opinion?
